testpilot/frontend/static-src/scripts/legal.js

```diff
@@ -0,0 +1,62 @@
+window.addEventListener('DOMContentLoaded', function (event) {
+  'use strict';
+
+  // Add an accordion to the Privacy Notice page
+  if (location.pathname === '/privacy') {
+    var ps = document.querySelectorAll('h2 ~ p');
+    var tablist = ps[0].parentElement;
```

Comment:
On [legal-docs](https://github.com/mozilla/legal-docs), we are using a [Python Markdown extension](https://pythonhosted.org/Markdown/extensions/attr_list.html) syntax to inject attributes, IDs and class names. I just checked the [remarkable](https://github.com/jonschlinkert/remarkable) library used on this textpilot repo but looks like it doesn't support such attribute injections unfortunately.

Since this Privacy Notice is pretty simple, I think it's safe to rely on JS to do tweaks.